cmp/chart/Chart.ts

```diff
@@ -26,6 +26,9 @@ import {DarkTheme} from './theme/Dark';
 import {LightTheme} from './theme/Light';
 import './Chart.scss';
 
+// Note that since this import is Typescript only, it is okay to import from the desktop package.
+import {UseContextMenuSpec} from '@xh/hoist/desktop/hooks';
```

Comment:
Think I might be inclined to either to just declare it as any, and forego the typechecking in this file.   Cool that this works, but avoid this weirdness altogether.  Hopefully our menus 2.0 won't have this problem.  :-)